hedera-node/hedera-app-spi/src/testFixtures/java/com/hedera/node/app/spi/fixtures/Assertions.java

```diff
@@ -46,4 +47,29 @@ public static void assertThrowsPreCheck(
     public interface PreCheckRunnable {
         void run() throws PreCheckException;
     }
+
+    /**
+     * Asserts that the given {@code runnable}, when run, throws a {@link HandleException} with the given
+     * expected {@link ResponseCodeEnum}.
+     *
+     * @param runnable The runnable which will throw a {@link HandleException}.
+     * @param expected The expected status code of the exception
+     */
+    public static void assertThrowsHandle(
+            @NonNull final HandleRunnable runnable, @NonNull final ResponseCodeEnum expected) {
+        try {
+            runnable.run();
+            throw new AssertionError("Expected " + expected + " but no exception was thrown");
+        } catch (final HandleException actual) {
+            if (!actual.getStatus().equals(expected)) {
+                throw new AssertionError("Expected " + expected + " but got " + actual);
+            }
+        }
+    }
+
```

Comment:
I think we have something called `validateTrue` in `HandleException`. We can use it. 
For eg: 
```
 validateTrue(treasury.isPresent(), INVALID_TREASURY_ACCOUNT_FOR_TOKEN);
```

Reply:
Was thinking something like 
```
 assertThatThrownBy(() -> subject.handle(.....))
                .isInstanceOf(HandleException.class)
                .has(responseCode(INVVALID_TOKEN_ID));`
Just preference.

Reply:
Done. I created a new condition called `handleResponseCode(...)` because `responseCode(...)` only looks at `PreCheckException`